### DISCUSSION TITLE
How to change cursor color in insert mode?

### DISCUSSION THREAD

**Discussion by Yevgnen** [helix-editor/helix#3409] [Q&A]
I'm tweaking a theme to use white background, and try the make the bar cursor back in insert mode otherwise the cursor becomes invisible. But after I checked some of the builtin themes, the cursor color of insert mode seems to always be white. 

For example, I see that settings `'ui.cursor.insert' = { fg = 'orangeN', bg = 'orangeN' }` are in `pop-dark.toml`. But the cursor is still white.

<img width="476" alt="Screen Shot 2022-08-13 at 15 49 16" src="https://user-images.githubusercontent.com/16749790/184474547-65c8125f-23a4-4ab9-a748-c0ea2156ab78.png">
<img width="466" alt="Screen Shot 2022-08-13 at 15 44 35" src="https://user-images.githubusercontent.com/16749790/184474516-686632a9-85f8-45c8-aee3-a9687b921e0c.png">
<img width="505" alt="Screen Shot 2022-08-13 at 15 45 09" src="https://user-images.githubusercontent.com/16749790/184474520-885e9521-6f2d-4315-9c28-e95a78bc823f.png">

How can I change that?

**Comment by CptPotato** [comment]
I *think* the bar cursor's color can only be set by the terminal, not helix.

Anyone correct me if I'm wrong.

**Reply by Yevgnen** [reply]
Thanks for the information.
